src/components/admin/app-navbar/AppNavbar.vue

```diff
@@ -17,57 +30,61 @@
             hello@epicmax.co
           </a>
         </span>
-        <div>
-          <va-button
-            href="https://github.com/epicmaxco/vuestic-admin"
-            color="#000000"
-            class="app-navbar__button text--bold"
-            icon="fa fa-github"
-            target="_blank"
-          >
-            {{$t('navbar.support')}}
-          </va-button>
-        </div>
+        <va-button
+          href="https://github.com/epicmaxco/vuestic-admin"
+          color="#000000"
+          class="app-navbar__button"
+          icon="fa fa-github"
+          target="_blank"
+        >
+          {{$t('navbar.support')}}
+        </va-button>
       </div>
-    </template>
-    <div class="app-navbar__actions row flex-nowrap align--center">
-      <color-dropdown class="va-navbar__item"/>
-      <message-dropdown class="va-navbar__item"/>
-      <notification-dropdown class="va-navbar__item"/>
-      <!--<settings-dropdown-->
-        <!--class="va-navbar__item"-->
-        <!--:navbar-view.sync="navbarViewProxy"-->
-      <!--/>-->
-      <language-dropdown class="va-navbar__item"/>
-      <profile-dropdown class="va-navbar__item">
-        <span>{{userName}}</span>
-      </profile-dropdown>
+      <app-navbar-actions
+        class="app-navbar__actions md5 lg4"
+        :user-name="userName"
+      />
     </div>
-  </va-navbar>
+    <div
+      class="app-navbar__shape"
+      :style="shapeStyle"
+    ></div>
+  </nav>
 </template>
 
 <script>
-import VaIconVuestic
-  from 'vuestic-ui/src/components/vuestic-components/va-icon/va-iconset/VaIconVuestic'
-import HeaderSelector from './components/HeaderSelector'
-import LanguageDropdown from './components/dropdowns/LanguageDropdown'
-import ProfileDropdown from './components/dropdowns/ProfileDropdown'
-import NotificationDropdown from './components/dropdowns/NotificationDropdown'
-import MessageDropdown from './components/dropdowns/MessageDropdown'
-import SettingsDropdown from './components/dropdowns/SettingsDropdown'
-import ColorDropdown from './components/dropdowns/ColorDropdown'
+import VaIconVuestic from '../../../iconset/VaIconVuestic'
+import AppNavbarActions from './components/AppNavbarActions'
+import { hex2hsl } from '../../../services/color-functions'
+
+// TODO: need to update for all color-functions
```

Comment:
Maybe makes sense to finish that in scope of current PR?

Reply:
Yeap. It's a good idea!